### ISSUE TITLE
pistache.io website is throwing an SSL error.

### ISSUE THREAD

**Issue by baylf2000** [pistacheio/pistache#977]
The pistache.io website is throw a `NET::ERR_CERT_COMMON_NAME_INVALID` error. Confirmed from multiple browsers.

![image](https://user-images.githubusercontent.com/1748322/133177900-7be7d6aa-c27a-4a1c-b4cf-0728579e7030.png)

**Comment by Tachi107** [comment]
Yes, https is not enabled in the repo's settings. Only the ones with write permissions can change that setting.
